core/src/parking_lot.rs

```diff
@@ -16,6 +16,39 @@ use core::{
 use smallvec::SmallVec;
 use std::time::{Duration, Instant};
 
+cfg_if! {
+    if #[cfg(target_arch = "wasm32")] {
```

Comment:
This should only be used on wasm32-unknown-unknown. Other wasm targets have a working `Instant` type which should be used.

Reply:
Yep! Changed it.